src/components/DevicesView.jsx

```diff
@@ -38,7 +38,9 @@ const DevicesView = ({ t, f, isFetching, devices, openDeviceRevokeModale, device
         </div>
         <div className={classNames(tableStyles['coz-table-body'], tableStyles['set-table-devices'])}>
           <div>
-            {devices.map(device => (
+            {devices
+            .filter(device => ['mobile', 'desktop'].indexOf(device.client_kind) != -1)
```

Comment:
`Expected '!==' and instead saw '!='``

![](http://www.reactiongifs.com/r/trrfd.gif)